Tests/LinkCommandsTest/delete_script.txt

```diff
@@ -0,0 +1,5 @@
+find . -type f -not \( -name '*o' -or -name '*json' -or -name 'ade' -or -name ippicv \) -delete
```

Comment:
Could you please move it out to this repository https://github.com/Software-Analysis-Team/link-commands